src/i18n/de/ui.ts

```diff
@@ -58,16 +71,54 @@ export default UIDictionary({
 	'aside.tip': 'Tipp',
 	'aside.caution': 'Achtung',
 	'aside.danger': 'Gefahr',
+	// `<LanguageSelect>` vocabulary
+	'languageSelect.label': 'Sprache auswählen',
+  	// Integrations vocabulary
+	'integrations.changelog': 'Changelog',
+	'integrations.footerTitle': 'Weitere Integrationen',
+	'integrations.renderers': 'UI-Frameworks',
+	'integrations.adapters': 'SSR-Adapter',
+	'integrations.others': 'Sonstiges',
+  	// Checklist component
+	'checklist.or': 'oder',
+  	// Multiple Choice component
+	'multipleChoice.defaultCorrect': 'Korrekt!',
+  	'multipleChoice.defaultIncorrect': "Versuch's nochmal!",
+  	'multipleChoice.submitLabel': 'Absenden',
+  	// Tutorial Progress
+	'progress.todo': 'To-do',
+  	'progress.done': 'Erledigt',
+  	// Tutorial Navigation
+  	'tutorial.trackerLabel': 'Tutorial Tracker',
+	'tutorial.unit': 'Teil',
+  	// Tutorial
+	'tutorial.getReady': 'Mach dich bereit,…',
 	// Feedback Fish widget
 	'feedback.button': 'Gib uns Feedback',
 	'feedback.a11yLabel': 'Feedback-Formular',
 	'feedback.formTitle': 'Wie kann ich helfen?',
```

Comment:
```suggestion
	'feedback.formTitle': 'Worum geht es?',
```

Da ich nicht wusste, wer "ich" sein soll, habe ich mal eine andere Formulierung gewählt. Mir ist bewusst, dass du das nicht geändert hast, aber es ist mir im Kontext aufgefallen.